content/pt-br/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,58 @@
+- 
```

Comment:
```suggestion
---
```

Reply:
Thanks, @brendavsantos, The build is broken because of the yaml lines that are missing, could you update them?

Reply:
Done!